oap-logstream-data-object/src/main/java/oap/logstream/data/object/BinaryObjectLogger.java

```diff
@@ -49,7 +48,11 @@
 import static java.util.Objects.requireNonNull;
 import static oap.template.ErrorStrategy.ERROR;
 
-public class BinaryObjectLogger extends DataModelTemplate {
+public class BinaryObjectLogger {

```

Comment:
Am I right, it was handful to use `TemplateEngine` from `DataModelTemplate`, but `BinaryObjectLogger` isn't `DataModelTemplate`, right?

Reply:
BinaryObjectLogger = TemplateEngine + Logger ( all in one )